test/ibmq/runtime/test_utils.py

```diff
@@ -0,0 +1,31 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for runtime utils."""
+
+from qiskit.providers.ibmq.runtime.utils import ProviderRequestParams
+from qiskit import QuantumCircuit
+
+
+from ...ibmqtestcase import IBMQTestCase
+
+
+class TestProviderRequestParams(IBMQTestCase):
+    """ Tests the correctness of the ProviderRequestParams class """
+
+    def test_provider_request_params(self):
```

Comment:
This is a runtime related change so test should be added to `test_runtime`. Also test should verify that the server gets the correct input parameters.

Reply:
There is a validation step now that the user can take, but will also be run when calling `IBMRuntimeService.run()`